src/config/settings.py

```diff
@@ -253,6 +253,11 @@ def alt_or_cmd(self) -> str:
     # Switch to the last opened page/chat, similar to Alt+Tab on most desktops.
     last_page = ["Ctrl+Tab"]
 
+    # Go throgh history of opened chats,
+    # similar to the "page back" and "page forward" keys in web browsers
+    visit_history_back = ["Ctrl+H"]
+    visit_history_forward = ["Ctrl+Y"]
```

Comment:
Ah, see I forgot that `Z` and `Y` are swapped on a lot of keyboard layouts, so this is indeed not a good default binding (I'll use it myself anyway). `Ctrl+Shift+H` is better. In fact, if we want to make it intuitive, we could even make it `Alt+Left` and `Alt+Right` like in browsers; and maybe even mouse button 4 and mouse button 5.

About the setting names; I was trying to show that it's about the history of visited rooms, rather than history of received messages etc. but I suppose the comment also does it justice, so I'll shorten the names as you suggested.

Reply:
Alt+Left/Right are already used for horizontal scrolling (e.g. in the image viewer), but extra mouse controls sound good. This can be done using [`TapHandler`s](https://doc.qt.io/qt-5/qml-qtquick-taphandler.html) with `acceptedButtons` set to one of the [supported buttons](https://doc.qt.io/qt-5/qt.html#MouseButton-enum).

How about `earlier_page` and `later_page` for the settings name? That's consistent with the `last_page` too